lib/createOrUpdatePullRequest.js

```diff
@@ -51,6 +51,7 @@ export default async function createOrUpdatePullRequest({
 
   await exec("git", ["config", "user.name", author]);
   await exec("git", ["config", "user.email", email]);
+  await exec("git", ["add", "package.json"]);
   await exec("git", ["add", "package-lock.json"]);
```

Comment:
[suggestion] Can we reduce the call of `exec` once?

```suggestion
  await exec("git", ["add", "package.json", "package-lock.json"]);
```